msteams-platform/concepts/build-and-test/share-to-teams.md

```diff
@@ -57,11 +57,11 @@ The following image displays the Share-to-Teams pop-up experience:
     </div>
     ```
 
-1. If your page dynamically renders content, you can use the the `shareToMicrosoftTeams.renderButtons()` method to force the **Share** button to render at the appropriate place in the pipeline.
+1. If your page dynamically renders content, you can use the `shareToMicrosoftTeams.renderButtons()` method to force the **Share** button to render at the appropriate place in the pipeline.
```

Comment:
If your page dynamically renders content, you can use the `shareToMicrosoftTeams.renderButtons()` method to force **Share** to render at the appropriate place in the pipeline.

Reply:
Fixed.